go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/giantswarm/k8scloudconfig/v6
+module github.com/giantswarm/k8scloudconfig/v7
```

Comment:
I need to add this to the action

Reply:
You also need to update the local module https://github.com/giantswarm/k8scloudconfig/blob/bd023cb0c0cca21f8b6729dd01f1c1852d6c1aef/pkg/template/cloudconfig.go#L15 and the sibling test file which still reference v6 and cause CI to fail.